### ISSUE TITLE
Accessibility issue: insufficient contrast for text links

### ISSUE THREAD

**Issue by coliff** [h5bp/html5boilerplate.com#394]
**I'm submitting a ...**

- [x] bug report
- [ ] feature request
- [ ] other (Please do not submit support requests here (below))

Run `npx pa11y https://html5boilerplate.com/`

```
Welcome to Pa11y

 > Running Pa11y on URL https://html5boilerplate.com/

Results for URL: https://html5boilerplate.com/

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.89:1. Recommendation:  change text colour to #005c8a.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── html > body > section > div > ul > li:nth-child(1) > a
   └── <a href="https://github.com/h5bp/html5-boilerplate/issues"> <img src="github.s...</a>

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.89:1. Recommendation:  change text colour to #005c8a.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── html > body > section > div > ul > li:nth-child(2) > a
   └── <a href="https://stackoverflow.com/questions/tagged/html5boilerplate"> <img src="stack-ov...</a>
```
